src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -2351,86 +2343,72 @@ namespace Akka.Dispatch
         Akka.Actor.Settings Settings { get; }
     }
     public interface IMultipleConsumerSemantics : Akka.Dispatch.ISemantics { }
+    public interface IProducesMessageQueue<TQueue>
```

Comment:
New `IProducesMessageQueue<TQueue>` interface - acts as the compliment for the existing `IRequiresMessageQueue<TQueue>` type implemented by @cconstantin. This type checking is now actually implemented inside the `Mailboxes` class.